README.md

```diff
@@ -1,38 +1,67 @@
+
+Our repository doesn't allow you to make changes directly to the `main` branch. Create a working branch and make pull requests from your fork to [rancher/rancher-docs](https://github.com/rancher/rancher-docs). 
+
+## Style & Formatting
+
+The docs are written in [Markdown](https://www.markdownguide.org/cheat-sheet/). We refer to the Microsoft [style guide](https://learn.microsoft.com/en-us/style-guide/welcome/) and generally use standard American English. Many pages are also available in Standard Chinese. We plan to add more language support.
+
+Every docs page contain metadata in the first few lines:
+
+```
+---
+title: Some Title
+---
+```
+
+The `title` is rendered as the page's headline. The renderer wraps the `title` value in H1 tags, which are equivalent to `#` in Markdown syntax. This means that all subsequent headers on the page should be second level (`##`) or more.
+
+## Organization
+
+Directories in the repo correspond to submenus in the site sidebar. We try to keep our submenus to a maximum of three levels deep, or four if absolutely necessary.
+
+The sidebar on the live site is rendered based on the contents of the file, `sidebars.json`, which is located in the top level of the repository. If you add, move, or delete a page, `sidebars.json` must be updated.
+
+## Docs Website
 
 The Rancher Docs website is built with [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
 
-## Installation
+You can run the site on your local machine, to preview how pages on your working branch will look live.
 
-The Rancher Docs repository already contains a yarn.lock file, which contains the dependencies you need to build the website.
+First, install Docusaurus 2:
 
 1. If you haven't already, install [Node](https://nodejs.org/en/download/) and [Yarn](https://yarnpkg.com/getting-started/install).
-1. Fork and clone the rancher-docs repository.
 1. Go into your local rancher-docs folder.
-1. Run `yarn` to install Docusaurus and associated dependencies.
+1. The Rancher Docs repository already contains a yarn.lock file, which contains the dependencies you need to build the website. Run `yarn` to install Docusaurus and associated dependencies.
 
-## Local Development
+### Start Site
 
 ```
 yarn start
 ```
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
-
-You can also use Docker to launch the website without needing to install and configure Yarn:
+This command starts a local development server for Docusuarus 2, and opens up a browser window. Most changes are reflected live without having to restart the server.
 
-```
-docker run --rm -it -v $PWD:$PWD -w $PWD -p 3000:3000 node yarn start -h 0.0.0.0
-```
+Note: The `yarn start` command won't include some important static site features. For example, the site will lack versioning for different languages. If you need these features, use `yarn build`.
```

Comment:
Slight adjustment needed on the example. The ability to have multiple languages simultaneously (e.g. switching using the language dropdown) in the dev environment is not available, not the versioning of languages. Side note, you can still preview/access different languages independently in the dev environment if you pass `--locale [LOCALE]` to `yarn start`.